projects/storefrontlib/src/lib/user/components/register/register.component.ts

```diff
@@ -64,6 +67,7 @@ export class RegisterComponent implements OnInit, OnDestroy {
       .pipe(
         switchMap(data => {
           if (data && data.access_token) {
+            this.globalMessageService.remove(GlobalMessageType.MSG_TYPE_ERROR);
```

Comment:
if we would like to delete login/register-related error messages only, we should pass a second parameter to this function with an index of the message. FYI @marlass

Reply:
I think we should remove everything. And we really can't know which one. We would need to check text content of them.